triciaswift/tastevine-server
# Added create method for recipe view

I added a create method on Recipe View

Supported routes
`/recipes` for a POST request will return a single object of the newly created recipe

## Steps to test
1. Pull down the `ts-create-recipe` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Use this token in the header:
```
Token c29b168a0c1e461b3ff36dd9ce15e6e1d1768597
```
5. Perform a POST to `http://localhost:8000/recipes`  with the following body:
```
{
    "title": "Better than Olive Garden Fettuccine Alfredo",
    "instructions": "Melt butter just under medium heat. Add minced garlic and cook for 3 minutes. Stir often and make sure the garlic doesn't brown. | Add heavy whipping cream. Whisk until the butter and cream come together. It will seem like they repel each other but keep whisking. | Once blended let thicken for about 10 minutes. Whisking often. Add grated parmesan cheese and black pepper. Whisk and let cook until all the cheese has melted, about 5 minutes. | Prepare fettuccini noodles to al dente, strain and serve.",
    "image": null,
    "ingredients": [
        {
      "ingredient": 1,
      "quantity": "2",
      "unit": "c"
    },
    {
      "ingredient": 2,
      "quantity": "3",
      "unit": ""
    }
    ],
    "categories": [5]
}
```
6. Should receive a 200 status and receive an object back with the newly created information.
